field.py

```diff
@@ -64,11 +64,11 @@ def SetFieldRadioButtons(self):
         
         self.NewField(self.Vfield, self.Vfield.value)  # setChecked the current field
```

Comment:
yes, this passed unnoticed. I did notice it in `tilt.py` and there I set as True, since it is intended to do the same as an ordinary change, i.e. check the radio button.
However, as I said, the difference between strong and weak is somewhat subjective. To avoid that, I chose "avoid costly computation" as the ultimate purpose of a weak change. As this is a direct call and `field.py` don't make any difference between strong and weak, the best would be to stay with the standard  `True`.